modules/deploy/partials/cloud/controlplane-api.adoc

```diff
@@ -0,0 +1,288 @@
+:tag-clusters: api:ROOT:cloud-api.adoc#tag--ClusterService
+:tag-networks: api:ROOT:cloud-api.adoc#tag--NetworkService
+:tag-operations: api:ROOT:cloud-api.adoc#tag--OperationService
+:tag-resource-groups: api:ROOT:cloud-api.adoc#tag--ResourceGroupService
+:tag-serverless-regions: api:ROOT:cloud-api.adoc#tag--ServerlessRegionService
+:tag-serverless-clusters: api:ROOT:cloud-api.adoc#tag--ServerlessClusterService
+
+The Redpanda Cloud API is a collection of REST APIs that allow you to interact with different parts of Redpanda Cloud. The Control Plane API enables you to programmatically manage your organization's cloud environment outside of the Redpanda Cloud UI. You can call the API endpoints directly, or use tools like Terraform or Python scripts to automate cluster management.
+
+You can also use the Cloud API with Redpanda Serverless. See the xref:deploy:deployment-option/cloud/serverless.adoc[Serverless] documentation to learn about the limited availability (LA) release, including cluster limits, trial credits, and supported features.
```

Comment:
You could conditionalize this so it only appears on the Serverless page.

Reply:
@vladoschreiner same question as above, do we only want to show Serverless content or mention it if the reader is viewing the doc URL `current/deploy/deployment-option/cloud/api/cloud-serverless-controlplane-api/`? Or is it OK for BYOC/Dedicated readers to see some mention of Serverless? 

We could do that and also perhaps update the API [Quickstart](https://docs.redpanda.com/current/deploy/deployment-option/cloud/api/cloud-api-quickstart/) and/or [Overview](https://docs.redpanda.com/current/deploy/deployment-option/cloud/api/cloud-api-overview/) instead so we redirect Serverless readers from those doc entry points. Let me know what you think. cc @towfiqa

Reply:
What's the purpose of this paragraph? Remove please.